CMakeLists.txt

```diff
@@ -32,12 +32,6 @@ unset(${SPECTRE_NAME}_VERSION_PATCH)
 if (CMAKE_VERSION VERSION_GREATER_EQUAL 3.19)
```

Comment:
what do you mean by "now"? Which version of CMake?

Reply:
Updated the commit message. (CMake policies work with the `cmake_minimum_required(VERSION)`, which is 3.18 for us at the moment. Any policy introduced before that uses the NEW behavior by default, and any policy introduced after uses OLD. CMP0094 was added in v3.15, so when we upgraded to v3.18 it became obsolete to toggle it to NEW.)